freeipa/src/main/resources/schema/app/20200228133843_CDPCP-1533_add_operation_to_usersync_status.sql

```diff
@@ -0,0 +1,24 @@
+-- // CDPCP-1533 Add Operation to usersync status. The lastfullsyncstarttime
+-- // and lastfullsyncendtime columns dropped because they are redundant with
+-- // the information stored in lastrequestedfullsync and lastsuccessfulfullsync.
+
+ALTER TABLE usersyncstatus
+    ADD COLUMN IF NOT EXISTS lastrequestedfullsync_id bigint
+        CONSTRAINT fk_usersyncstatus_lastrequestedoperationid
+        REFERENCES operation,
+    ADD COLUMN IF NOT EXISTS lastsuccessfulfullsync_id bigint
+        CONSTRAINT fk_usersyncstatus_lastsuccessfuloperationid
+        REFERENCES operation,
+    DROP COLUMN IF EXISTS lastfullsyncstarttime,
+    DROP COLUMN IF EXISTS lastfullsyncendtime;
```

Comment:
this will break the schema during rolling release. I will send you the docs in private